content/blogposts/2023/open-sourcing-cody.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Open sourcing Cody"
+authors:
+  - name: Beyang Liu
+    url: https://twitter.com/beyang
+publishDate: 2023-03-27T10:00-07:00
+description: "Cody, Sourcegraph's AI-enabled editor assistant, is now open source"
+tags: [blog]
+slug: 'open-sourcing-cody'
+published: true
+heroImage: https://raw.githubusercontent.com/sourcegraph/sourcegraph/main/client/cody/resources/cody.png
+socialImage: https://raw.githubusercontent.com/sourcegraph/sourcegraph/main/client/cody/resources/cody.png
+---
+
+<p><iframe width="100%" height="415" src="https://www.youtube.com/embed/EKCtoU0oiCU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe></p>
+
+We've open-sourced Cody, Sourcegraph's AI-enabled editor assistant under the Apache 2.0 license. You can view the code [here](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/client/cody) and you can join our [livestreams](https://www.twitch.tv/sourcegraph) where we'll provide a way to onboard to the codebase and build new features in public. If you'd like to try it out, follow [these instructions](https://docs.sourcegraph.com/cody#install-cody).
```

Comment:
Minor - “here” is not good link text, make the link text descriptive of what it takes you to